utils/spacewalk-utils.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jun 13 18:00:00 CEST 2022 - philippe.bidault@getronics.com
+
+- version 4.3.11-1
+  * Add EPEL8 for Almalinux 8 and Rocky 8 in spacewalk-common-channels.ini
```

Comment:
```suggestion
- Add EPEL8 for Almalinux 8 and Rocky 8 in spacewalk-common-channels.ini
```
And lines 1-4 can be removed :-)

Version, date, email etc... are added later, when we "tag" the packages